install-mac.sh

```diff
@@ -119,13 +152,26 @@ fi
 fancy_echo "Installing Docker"
 if [ ! -d "/Applications/Docker.app" ]
 then
-    curl -Lo Downloads/Docker.dmg  https://download.docker.com/mac/stable/Docker.dmg
-    open Downloads/Docker.dmg
+    curl -Lo ~/Downloads/Docker.dmg  https://download.docker.com/mac/stable/Docker.dmg
+    sudo hdiutil attach  -mountpoint <path-to-desired-mountpoint> ~/Downloads/Docker.dmg
```

Comment:
1) You missed replacing the actual value. <path-to-desired-mounpoint>
2) When you open a dmg it automatically mounts